opts.go

```diff
@@ -97,6 +97,8 @@ type ParsedOpts interface {
 	RunFatal()
 	//Selected returns the subcommand picked when parsing the command line
 	Selected() ParsedOpts
+	//Children returns the subcommands of this command
+	Children() map[string]Opts
```

Comment:
hey gary, sorry for the delay - very slow on my opensource stuff atm ☹️ should this be a map of `ParsedOpts` ?

Allowing the user to get back `Opts` would allow another `Parse` which is currently undefined...